jest.config.js

```diff
@@ -5,7 +5,9 @@ module.exports = {
   testEnvironment: 'node',
   testPathIgnorePatterns: ['.d.ts', '.js', '__mocks__'],
   moduleNameMapper: {
+    '^@shapeshiftoss\\/caip': '@shapeshiftoss/caip',
     '^@shapeshiftoss\\/logger': '@shapeshiftoss/logger',
+    '^@shapeshiftoss\\/types': '@shapeshiftoss/types',
     '^@shapeshiftoss\\/([^/]+)': '@shapeshiftoss/$1/src',
```

Comment:
My recommendation would be to use a different prefix for internal packages

Reply:
Everything is technically being published at this point, so the `shapeshiftoss` scope name is required. The module mapping was done specifically so the `__mocks__` could be referenced correctly, so maybe there is a more correct way to do that piece to clean this up.

Reply:
Found a better way to handle both cases: `'^@shapeshiftoss\\/([^/]+)': ['@shapeshiftoss/$1/src', '@shapeshiftoss/$1'],`